### PR TITLE
Add project skeleton for the SQL API Extension

### PR DESCRIPTION
**Pull request by dimas-b**
Initial POMs for the SQL module and it parent

### CODE REVIEW

**Comment by dimas-b** [comment]
@dougwettlaufer : please have a looks at the GH/build stuff... I'm not sure I caught up with all the latest changes in that area that you made in the `stargate` repo :)

File: pom.xml [hunk 1]
```diff
@@ -0,0 +1,86 @@
...
+        <module>sql</module>
+    </modules>
+
+    <properties>
+        <project.build.sourceEncoding>UTF-8</project.build.sourceEncoding>
+    </properties>
+
+    <repositories>
+        <repository>
+            <id>github</id>
```
Comment (dougwettlaufer) [inline pom.xml:26]:
```suggestion
            <id>gh</id>
```

File: sql/pom.xml [hunk 2]
```diff
@@ -0,0 +1,72 @@
...
+    <groupId>io.stargate.sql</groupId>
+    <artifactId>avatica-sql</artifactId>
+
+    <properties>
+        <junit.version>5.6.2</junit.version>
+    </properties>
+
+    <repositories>
+        <repository>
+            <id>github</id>
```
Comment (dougwettlaufer) [inline sql/pom.xml:22]:
```suggestion
            <id>gh</id>
```

File: sql/pom.xml [hunk 3]
```diff
@@ -0,0 +1,72 @@
...
+         xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+    <modelVersion>4.0.0</modelVersion>
+
+    <parent>
+        <groupId>io.stargate.api</groupId>
+        <artifactId>api-extensions</artifactId>
+        <version>0.0.1-SNAPSHOT</version>
+    </parent>
+
+    <groupId>io.stargate.sql</groupId>
```
Comment (dougwettlaufer) [inline sql/pom.xml:13]: Should we group all these under `io.stargate.api`? So `io.stargate.api.sql`.

**Review by dougwettlaufer — DISMISSED** [review]

File: sql/pom.xml (hunk 3 above)
Comment (dimas-b) [reply sql/pom.xml]: Renamed. Also, renamed the java code package to `io.stargate.api.sql`

**Review by dougwettlaufer — APPROVED** [review]

**Pull request merged**